src/TransformationValidator.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\TransformationMigrate;
+
+use Keboola\TransformationMigrate\Exception\CheckConfigException;
+
+class TransformationValidator
+{
+    private array $config;
+
+    public function __construct(array $config)
+    {
+        $this->config = $config;
+    }
+
+    public function validate(): void
+    {
+        $this->validateBackend();
+        $this->validatePhases();
+    }
+
+    private function validateBackend(): void
+    {
+        $backends = array_map(
+            fn(array $v) => sprintf('%s-%s', $v['configuration']['backend'], $v['configuration']['type']),
+            $this->config['rows']
+        );
+        $uniqueBackends = array_unique($backends);
+
+        if (count($uniqueBackends) > 1) {
+            throw new CheckConfigException(sprintf(
+                'Transformations in the bucket "%s" don\'t have the same backend.',
+                $this->getTransformationName()
+            ));
+        }
+    }
+
+    private function validatePhases(): void
+    {
+        $phases = array_map(fn(array $v) => $v['configuration']['phase'], $this->config['rows']);
+        $uniquePhases = array_unique($phases);
+
+        if (count($uniquePhases) > 1) {
+            throw new CheckConfigException(sprintf(
+                'Transformations in the bucket "%s" don\'t have the same phase.',
```

Comment:
Navrhujem pridat `uniquePhases` do vypisu, bude tam o uzitocnu informaciu viac. Ked to spustis napr. hromadne tak rovno z vystupu uvidis, ake kombinacie backendov tam este su.

Reply:
to vidíš na první pohled z detailu transformace, ale přidat to tam můžeme

Reply:
hromadně migraci nespustíš... půjde to jen po jednotlivých transformacích

Reply:
jj, ale predpokladam, ze bude na to nejaky script, ktory to bude spustat na jednotlivych transformaciach cez API.

Reply:
alebo to budu migrovat iba uzivatelia rucne?

Reply:
hodilo by sa mat v Jire link na dokument, kde je to popisane

Reply:
ne, na to bude jen tlačítko v UI a klienti to budou dělat ručně